activerecord/CHANGELOG.md

```diff
@@ -1735,5 +1735,9 @@
 
     *Jonathan Hefner*
 
+*   Fix connections being left in open transactions.
+
+    *Nick Dower*
```

Comment:
We add new entries to the top of the CHANGELOG.

Perhaps this can be expanded a bit more to mention the circumstances where the transactions were left open?

Reply:
Thank you for the help. I have moved the comment to the top of the file and added more detail. Please let me know what you think.

Reply:
Looks great to me!!